src/babylon.node.ts

```diff
@@ -151,14 +151,7 @@
                 }
                 this._parentNode._children.push(this);
 
-                if (!previousParentNode) {
-                    // Need to remove from rootNodes
-                    const rootNodeIndex = this._scene.rootNodes.indexOf(this);
-
-                    if (rootNodeIndex > -1) {
-                        this._scene.rootNodes.splice(rootNodeIndex, 1);
-                    }
-                }
+                if (!previousParentNode) this.removeFromSceneRootNodes();

```

Comment:
can you add {} and a line feed? (just cosmetic)

Reply:
would be nice in future to add tslint rules for such cases

Reply:
I agree, everything that could ensure coherent and automatic formatting is worth beeing added (tslin, prettier, pre-commit hook).

Another example, all the trailling whitespace should be removed they just add noise if your editor displays them and are likely to cause useless change in commit.

Reply:
I agree. If someone wants to volunteer to set it up in the project, I'll appreciate the help